src/mcode-pilot/utils/filterTreatmentData.js

```diff
@@ -149,23 +140,20 @@ function isSimilarPatient(treatmentDataPatient, similarPatientProps) {
                 // pathology
                 } else if (mcodeElement === 'onco.core.TumorMarkerTest') {
                     const receptorType = reference._tumorMarker._findingTopicCode.codeableConcept.coding[0].code.value;
-                    if (receptorType === '16112-5' && (!tumorMarkers.er || _.lowerCase(tumorMarkers.er) !== _.lowerCase(value))) {
+                    if (receptorType === '16112-5' && (!tumorMarkersLabeled["16112-5"] || tumorMarkersLabeled["16112-5"].value.code !== reference.findingResult._value._coding[0]._code.code)) {
```

Comment:
This should be a generic capability that can be used across tumormarkers without the need of hard coding values for specific types